hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestWatchForCommit.java

```diff
@@ -369,6 +369,9 @@ public void testWatchForCommitForGroupMismatchException() throws Exception {
       } catch (Exception e) {
         Assert.assertTrue(HddsClientUtils
             .checkForException(e) instanceof GroupMismatchException);
+        // client should not attempt to watch with
+        // MAJORITY_COMMITTED replication level
+        Assert.assertFalse(e.getMessage().contains("Watch-MAJORITY_COMMITTED"));
```

Comment:
Can you please explain what this is testing?  This assertion passes without any other code change on `master`.